### PR TITLE
Feature/81

### PR DESCRIPTION
**Pull request by saadmhybris**
#81 

@xak, @joseegm  - Please review. 

Let me know if any big feature is missing.

### CODE REVIEW

**Comment by saadmhybris** [comment]
@joseegm the conflict has been resolved.

File: docs/pages/guides/design-system-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+{: .docs-header-h2}
+
+Use the text utilities to set the text style and the justification.
+
+{% capture text-util %}
+<p class="fd-has-font-weight-bold">This text is bold</p>
+<p class="fd-has-font-style-italic">This text is italic</p>
+<p class="fd-has-text-align-center">This text is centered</p>
+<p class="fd-has-text-align-right">This text is right aligned</p>
+<p class="fd-text-transform-none">text with no transformation</p>
```
Comment (xak) [inline docs/pages/guides/design-system-cheat-sheet.md]: Please change this to match the others — `fd-has-text-transform-none`

File: docs/pages/guides/design-system-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+<p class="fd-has-font-family-header fd-has-font-weight-med">This is font with weight medium</p>
+<p class="fd-has-font-family-header fd-has-font-weight-semi">This is font with weight semi-bold</p>
+<p class="fd-has-font-family-header fd-has-font-weight-bold">This is font with weight bold</p>
+{% endcapture %}
+{% include display-example.html component=type-weight %}
+
+## Colors
+{: .docs-header-h2}
+
+All colors on the [color pallet](colors.html). Can be accessed via SCSS function and respective CSS classes.
```
Comment (xak) [inline docs/pages/guides/design-system-cheat-sheet.md]: sp. — palette

File: docs/pages/guides/design-system-cheat-sheet.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
+---
+title: Design System Utilities
+keywords: design system cheat sheet
+sidebar: left-navigation-sidebar
+toc: false
+permalink: design-system-utilities.html
+folder: guides
+summary: The following guide covers all the major utility features of the Fundamental UI (FDUI) design system toolkit. We recommend that you utilize these functions, mixins and helper classes to maintain consistency throughout your application. Using the toolkit features will ensure that you application will remain compatible and easy to upgrade with the future versions of FDUI, transition between themes and customization.
+---
```
Comment (xak) [inline docs/pages/guides/design-system-cheat-sheet.md]: Include a blurb here about preferably using the mixins and functions with a link. http://localhost:4000/how-to-use-scss.html#core-functions-and-mixins

**Pull request merged**